clients/extension/src/background/handlers/requestHandler.ts

```
@@ -10,6 +10,7 @@ import {
 } from '@core/chain/coin/ChainId'
 import { chainRpcUrl } from '@core/chain/utils/getChainRpcUrl'
 import { shouldBePresent } from '@lib/utils/assert/shouldBePresent'
+import { match } from '@lib/utils/match'
```

ESLint

[error] 'match' is defined but never used.

(@typescript-eslint/no-unused-vars)